operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1284,5 +763,516 @@ spec:
+                        2.7.0=registry.redhat.io/amq7/amq-streams-kafka-27-rhel7@sha256:f7f19ec9598c4c577460988f48ddf55c1103fe3883fbe4ea8813f2864f24662e
+                    - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
+                      value: registry.redhat.io/amq7/amq-streams-rhel7-operator@sha256:0e37f671e6d489c3e56ffb8f51f19a2a014b2cb3f57eafcc5e2931090932e725
+                    - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
+                      value: registry.redhat.io/amq7/amq-streams-rhel7-operator@sha256:0e37f671e6d489c3e56ffb8f51f19a2a014b2cb3f57eafcc5e2931090932e725
+                    - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
+                      value: registry.redhat.io/amq7/amq-streams-rhel7-operator@sha256:0e37f671e6d489c3e56ffb8f51f19a2a014b2cb3f57eafcc5e2931090932e725
+                    - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
+                      value: registry.redhat.io/amq7/amq-streams-bridge-rhel7@sha256:bed07186355a4c04ee50e90048e3bd3a023264093a9ef28f0278cce44640e93f
+                  livenessProbe:
+                    httpGet:
+                      path: /healthy
+                      port: http
+                    initialDelaySeconds: 10
+                    periodSeconds: 30
+                  readinessProbe:
+                    httpGet:
+                      path: /ready
+                      port: http
+                    initialDelaySeconds: 10
+                    periodSeconds: 30
+                  resources: {}
+          strategy:
+            type: Recreate
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: true
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  relatedImages:
+    - name: strimzi-cluster-operator
+      image: registry.redhat.io/amq7/amq-streams-rhel7-operator@sha256:0e37f671e6d489c3e56ffb8f51f19a2a014b2cb3f57eafcc5e2931090932e725
+    - name: strimzi-kafka-260
+      image: registry.redhat.io/amq7/amq-streams-kafka-26-rhel7@sha256:9413de5670950e85f43a5ee6d06940218f753f78f71489eb8f4fdcccdd21074e
+    - name: strimzi-kafka-270
+      image: registry.redhat.io/amq7/amq-streams-kafka-27-rhel7@sha256:f7f19ec9598c4c577460988f48ddf55c1103fe3883fbe4ea8813f2864f24662e
+    - name: strimzi-bridge
+      image: registry.redhat.io/amq7/amq-streams-bridge-rhel7@sha256:bed07186355a4c04ee50e90048e3bd3a023264093a9ef28f0278cce44640e93f
+  keywords:
+    - kafka
+    - messaging
+    - kafka-connect
+    - kafka-streams
+    - data-streaming
+    - data-streams
+    - streaming
+    - streams
+  labels:
+    name: strimzi-cluster-operator
+  links:
+    - name: Product Page
+      url: https://access.redhat.com/products/red-hat-amq
+    - name: Documentation
+      url: https://access.redhat.com/documentation/en-us/red_hat_amq/2020.q4/html-single/using_amq_streams_on_openshift/index
```

Comment:
Needs to be updated to the 1.7 link.

Reply:
@laidan6000 Do you know what downstream link we should put here?

Reply:
https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q2/html/using_amq_streams_on_openshift/index